### PR TITLE
Add monthCode for all userinfo files

### PR DESCRIPTION
**Pull request by Renucopata**

### CODE REVIEW

File: git-playground/pablo/userinfo [hunk 1]
```diff
@@ -1,4 +1,5 @@
 Pablo Antonio Huanca Hilari
 27th june 1989
 Systems engineering
-Santa Cruz de la Sierra
\ No newline at end of file
+Santa Cruz de la Sierra
+Pablo-07
```
Comment (moreHc) [inline git-playground/pablo/userinfo:5]: Pablo's birthday month is 06, please fix

**Review by moreHc — CHANGES_REQUESTED** [review]

**Review by moreHc — CHANGES_REQUESTED** [review]
All lines are missing "month code:"

**Comment by J0c4** [comment]
Hi @Renucopata , please don't forget to follow up pending changes for this PR

**Comment by Renucopata** [comment]
-Pablo's birthday month code fixed
-Add the word "month code: " to all user info files
-Add the rest of information on rene/userinfo file

**Review by J0c4 — APPROVED** [review]

**Comment by J0c4** [comment]
Changes look good.

@moreHc , since you requested changes (which is ok), PR needs your review/approval as well, please check and if everything looks ok, lets approve and merge

**Review by moreHc — APPROVED** [review]

**Pull request merged**